packages/component-library/src/index.js

```diff
@@ -25,5 +28,9 @@ export { default as Logo } from './Logo/Logo';
 export { default as ShareCollection } from './Share/ShareCollection';
 export { default as isClient } from './utils/isClient';
 export { default as CivicVictoryTheme } from "./VictoryTheme/VictoryThemeIndex";
+export { default as CollectionHero } from './Hero/CollectionHero'
+export { default as BaseMap } from './BaseMap/BaseMap';
+export { default as PullQuote } from './PullQuote/PullQuote';
```

Comment:
Looks like we'll either need to move `PullQuote` into its own directory, or update our import here to import it `from './PullQuote'`. We might want to do the former to be inline with the rest of our components.